src/App.tsx

```diff
@@ -3,34 +3,111 @@ import './App.scss';
 import Carousel from './components/Carousel';
 
 interface State {
-  images: string[];
+  itemWidth: number,
+  frameSize: number,
+  step: number,
+  animationDuration: number,
 }
 
+const images: string[] = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
 class App extends React.Component<{}, State> {
   state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    animationDuration: 1000,
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      itemWidth, frameSize, step, animationDuration,
+    } = this.state;
 
     return (
       <div className="App">
         {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+        <h1 data-cy="title">Carousel with {images.length} images</h1>
+
+        <div className="parametr-editor">
+          <label htmlFor="itemId">
+            Item width:
+
+            <input
+              id="widthId"
```

Comment:
```suggestion
          <label htmlFor="itemId">
            Item width:

            <input
              id="itemId"
```
htmlFoe and id in input should be same